spec/services/card_service_spec.rb

```diff
@@ -0,0 +1,159 @@
+        end
+      end
+
+      context 'when exit if first gets is exit' do
+        it do
+          expect(current_subject).to receive_message_chain(:gets, :chomp)
+          current_subject.destroy_card(accounts, current_account)
+        end
+      end
+
+      context 'with incorrect input of card number' do
+        it do
+          allow(current_subject).to receive_message_chain(:gets, :chomp).and_return(fake_cards.length + 1, 'exit')
+          expect { current_subject.destroy_card(accounts, current_account) }
+            .to output(/#{ERROR_PHRASES[:wrong_number]}/).to_stdout
+        end
+
+        it do
+          allow(current_subject).to receive_message_chain(:gets, :chomp).and_return(-1, 'exit')
+          expect { current_subject.destroy_card(accounts, current_account) }
+            .to output(/#{ERROR_PHRASES[:wrong_number]}/).to_stdout
+        end
+      end
+
+      context 'with correct input of card number' do
+        let(:accept_for_deleting) { 'y' }
+        let(:reject_for_deleting) { 'asdf' }
+        let(:deletable_card_number) { 1 }
+        let(:current_account) { Account.new }
+        let(:accounts) { [current_account] }
+
+        before do
+          current_account.instance_variable_set(:@card, fake_cards)
+        end
+
+        after do
+          File.delete(OVERRIDABLE_FILENAME) if File.exist?(OVERRIDABLE_FILENAME)
+        end
+
+        it 'accept deleting' do
+          commands = [deletable_card_number, accept_for_deleting]
+          allow(current_subject).to receive_message_chain(:gets, :chomp).and_return(*commands)
+
+          expect { current_subject.destroy_card(accounts, current_account) }.to change { current_account.card.size }
+            .by(-1)
+
+          expect(File.exist?(OVERRIDABLE_FILENAME)).to be true
+          file_accounts = YAML.load_file(OVERRIDABLE_FILENAME)
+          expect(file_accounts.first.card).not_to include(card_one)
+        end
+
+        it 'decline deleting' do
+          commands = [deletable_card_number, reject_for_deleting]
+          allow(current_subject).to receive_message_chain(:gets, :chomp).and_return(*commands)
+
+          expect { current_subject.destroy_card(accounts, current_account) }.not_to change(current_account.card, :size)
+        end
+      end
+    end
+  end
+
+  describe '#create_card' do
+    let(:current_account) { Account.new }
+    let(:accounts) { [current_account] }
+
+    context 'with correct outout' do
+      it do
+        CREATE_CARD_PHRASES.each { |phrase| expect(current_subject).to receive(:puts).with(phrase) }
+        allow(File).to receive(:open)
+        allow(current_subject).to receive_message_chain(:gets, :chomp) { 'usual' }
+
+        current_subject.create_card(accounts, current_account)
+      end
+    end
+
+    context 'when correct card choose' do
+      after do
+        File.delete(OVERRIDABLE_FILENAME) if File.exist?(OVERRIDABLE_FILENAME)
+      end
+
+      CARD_TYPES.each do |card_type|
```

Comment:
You don't include `spec/constants_for_spec.rb` 
How you have access to use this const 'CARD_TYPES'?

Reply:
It is connected by ```rspec``` because the end of the file is ```_spec.rb```